core/scripts/common/vrf/setup-envs/main.go

```diff
@@ -211,6 +212,7 @@ func main() {
 				coordinatorConfigV2,
 				*batchFulfillmentEnabled,
 				nodesMap,
+				true,
```

Comment:
This was already broken separately from a signature change. I'm not sure that true is the correct value.

Reply:
latest develop should have it fixed already, so better sync with develop

Reply:
Updated!